docs/pages/pmd/userdocs/extending/your_first_rule.md

```diff
@@ -127,7 +127,7 @@ copy-paste into your ruleset XML. The resulting element looks like so:
 <rule name="DontCallBossShort"
       language="java"
       message="Boss wants to talk to you."
-      class="net.sourceforge.pmd.lang.rule.xpath.XPathRule">
+      class="net.sourceforge.pmd.lang.rule.XPathRule">
```

Comment:
Actually, it is correct like it is: The class `XPathRule` moved with PMD 7 into the sub-package `xpath`.
So, what is currently documented on https://docs.pmd-code.org/latest/pmd_userdocs_extending_your_first_rule.html is correct for PMD 7.

(in PMD 6, it was without the sub-package `xpath`, e.g. https://docs.pmd-code.org/pmd-doc-6.55.0/pmd_userdocs_extending_your_first_rule.html).